### ISSUE TITLE
Drag-and-drop: text disappears, no changes indicated

### ISSUE THREAD

**Issue by sportflier** [RickStrahl/MarkdownMonster#210]
Markdown Monster Version 1.4:

I tried selecting some text in an existing document in the editor, then dragging and dropping this to another location in the document:

- The text disappears from where it was originally, but does not appear in the new location where I was dragging it to. The net effect is the text simply disappears rather than being moved to the new location. Not what I was expecting!
- After this happens, the application's save icon does not light up, indicating that MM does not think the document has unsaved changes.
- If I use undo, my text is restored, but selection is different from before (not a major problem but a bit odd).

**Comment by RickStrahl** [comment]
Unfortunately there's no support for drag and drop editing mostly due to the fact that internet explorer (Web Browser Control) doesn't allow for it to work properly.

ACE Editor supports it but it doesn't work with IE (https://ace.c9.io/ - check with IE) and when it comes to the Web Browser control ontop of that it gets downright strange (it looks like you're trying to drag an image when I try).

I've looked into this a bit, but I haven't found a way to make this work effectively. I can't even disable the D&D as the WPF control does some weird image type fixup.

However, I also don't see the behavior you're describing:  I can see something that looks like a drag operation taking place but it does nothing to the source document.

This seems to me like a low usage scenario. I realize many editors support this, but does anybody actually use drag and drop text editing? I don't think I've ever used that... 

There was a previous issue on this and I closed as not fixable for the moment.

**Comment by RickStrahl** [comment]
So I took another look and it turns I was wrong - drag and drop can work. The issue I had was that there was an unused drop handler in place that caused the drop to essentially eat the dropped text. I've fixed that and that now works. You can drag in Move and Copy modes.

Unfortunately, the UI for this is still terrible: For some reason I can't figure out dragging inside of the editor drags the entire editor as an opaque drag image while dragging. Everything works but the huge image being dragged is a bit distracting. Oddly when I run the form outside of the Web Browser environment Drag and Drop doesn't work at all in Internet Explorer - which is very strange. 

So, in short it now works (1.4.2+), but with 'UI artifacts'.
